loader.go

```diff
@@ -318,3 +337,64 @@ func RefreshEquipments(manager *DataManager, uri url.URL, connectionTimeout time
 	equipmentsLoadingDuration.Observe(time.Since(begin).Seconds())
 	return nil
 }
+
+func CallHttpClient(siteHost, token string ) (*http.Response, error){
+	client := &http.Client{}
+	data := url.Values{}
+	data.Set("query", "query($id: Int!) {area(id: $id) {vehicles{publicId, provider{name}, id, type, attributes ,latitude: lat, longitude: lng, propulsion, battery, deeplink } } }")
+	// Manage area id in the query (Paris id=6)
+	// TODO: load vehicles for more than one area (city)
+	area_id := 6
+	data.Set("variables", fmt.Sprintf("{\"id\": %d}", area_id))
+	req, err := http.NewRequest("POST", fmt.Sprintf("%s/v1?access_token=%s", siteHost, token), bytes.NewBufferString(data.Encode()))
+	req.Header.Set("content-type", "application/x-www-form-urlencoded; param=value")
+	if err != nil {
+		return nil, err
+	}
+	resp, err := client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	return resp, nil
+}
+
+func LoadFreeFloatingData (data *Data) ([]FreeFloating, error) {
+	// Read response body in json
+	freeFloatings := make([]FreeFloating, 0)
+	for i := 0; i < len(data.Data.Area.Vehicles); i++ {
+		ff, err := NewFreeFloating(data.Data.Area.Vehicles[i])
+		if err != nil {
+			return nil, err
+		}
+		freeFloatings = append(freeFloatings, *ff)
```

Comment:
```suggestion
             freeFloatings = append(freeFloatings, *NewFreeFloating(data.Data.Area.Vehicles[i]))
```
You should be able to construct a new FreeFloating object straight into you array.